USAGE_NOTES.md

```diff
@@ -1,5 +1,5 @@
 # USAGE NOTES
-## July 16th 2020
+## July 16th 2021
```

Comment:
Please clarify here what the migration is from and to what. For example, `upgrading from version 2020.8 to 0.0.3` or something similar.

Reply:
Done